### PR TITLE
Set Test Day topic using welcome message template (#57)

### PR DESCRIPTION
**Pull request by galgeek**
Hello @whimboo and @ashughes1!

Here's an update using a welcome message template for the Test Day topic. I copied from both Anthony's text and the current #qa channel topic; I hope it's ok!

I should be able to get this running shortly in #testdaybotTest.

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -329,15 +329,19 @@ client.addListener('pm', function(from, message) { // private messages to bot
...
             // if the start and end dates appear valid, set the test date
             if ((testDay.end > testDay.start) && (testDay.start > Date.now())) {
               if (timerID !== 0) {
                 clearTimeout(timerID);
               }
               timerID = setTimeout(updateTestDayData, testDay.start - Date.now());
-              client.say(from, "Next Test Day's start is " + testDay.start);
-              client.say(from, "Next Test Day's end is " + testDay.end);
+              client.say(from, "Next Test Day's start is " + testDay.start.toUTCString());
+              client.say(from, "Next Test Day's end is " + testDay.end.toUTCString());
```
Comment (whimboo) [inline bot.js]: That's another issue.

File: bot.js [hunk 2]
```diff
@@ -329,15 +329,19 @@ client.addListener('pm', function(from, message) { // private messages to bot
             testDay.start = new Date(command[1]);
             testDay.end = new Date(command[2]);
             testDay.etherpad = command[3];
-            testDay.topic = message.slice(message.indexOf(testDay.etherpad) + testDay.etherpad.length + 1);
+            testDay.topic = "Welcome to the Mozilla Quality Assurance channel. " +
+                            "Today we’re testing " +
+                            command.slice(4, cmdLen).join(" ") + ". " +
+                            "Please read " + testDay.etherpad + " for more " +
+                            "information and ask questions in this channel.";
```
Comment (whimboo) [inline bot.js]: I'm not sure if we want to keep around the while content, or only the real topic. I would prefer the latter, and update the setter for the topic with this template. @ashughes1 what would you prefer?

Comment (ashughes1) [inline bot.js]: > I'm not sure if we want to keep around the while content, or only the real topic. I would prefer the latter, and update the setter for the topic with this template. @ashughes1 what would you prefer?

Sorry, I do not understand the question.

Comment (whimboo) [inline bot.js]: Barbara already updated the code in https://github.com/galgeek/testdaybot/commit/e71347915df8ee898e44118b0e70729b18885677, so you were most likely looking at the new code. @galgeek please update the PR so the wording compares to what Anthony suggested some minutes ago on the issue.

**Comment by whimboo** [comment]
Squashed commit, and merged PR as https://github.com/mozilla/testdaybot/commit/36765bc1703d5a931872eabcc0f7cfe135753a4f.

**Pull request closed without merging**